src/file-picker/src/FilePicker.js

```diff
@@ -184,7 +188,20 @@ FilePicker.propTypes = {
    * Class name passed to the FilePicker.
    * Only use this if you know what you are doing.
    */
-  className: PropTypes.string
+  className: PropTypes.string,
+
+  /**
+   * Function that returns the call-to-action button text for selecting files.
+   * Gets the file count as number as parameter.
+   */
+  browseOrReplaceText: PropTypes.func,
+
+  /**
+   * Function that returns the text in the input field,
+   * Describing what file or files have been added.
+   * Gets the selected files as File[] as parameter.
```

Comment:
One last nitpick - can we update this to use the `@type` JSDoc annotation vs. describing it?

`* @type {(files: File[]) => string}`